src/Task/Context/GitCommitMsgContext.php

```diff
@@ -1,73 +1,42 @@
 <?php
 
+declare(strict_types=1);
+
 namespace GrumPHP\Task\Context;
 
 use GrumPHP\Collection\FilesCollection;
 
 class GitCommitMsgContext implements ContextInterface
 {
-    /**
-     * @var FilesCollection
-     */
     private $files;
```

Comment:
Removing the docblocks from properties is a bridge too far for the moment. Lets just keep them untill property types are introduced in the language.